src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -188,8 +281,8 @@ def to_json(self, content, pretty_print=False):
         logger.info ('To JSON using : pretty_print=%s ' % (pretty_print))
 
         return json_
-
-
+    
+    @retry(Exception, logger)
```

Comment:
Instead of `Exception` can check for perhaps `RequestException` 

https://github.com/kennethreitz/requests/blob/master/requests/exceptions.py#L13